### ISSUE TITLE
texlab references rustc

### ISSUE THREAD

**Issue by doronbehar** [NixOS/nixpkgs#95568]
**To Reproduce**

```
$ nix why-depends -f. texlab rustc
/nix/store/2wzyn5db2k9l97hhjifxwdxyk3srsl47-texlab-2.2.0
╚═══lib/libhtml2md.so: …IBC_2.2.5.GLIBC_2.14./nix/store/1r7l5brzhjjm1lnx04f4gszy1xgjm1gi-rustc-1.45.0/lib/rustlib/x86_64…
    => /nix/store/1r7l5brzhjjm1lnx04f4gszy1xgjm1gi-rustc-1.45.0
```

**Expected behavior**
No rustc reference

**Notify maintainers**

@MetaDark & myself.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: texlab
# a list of nixos modules affected by the problem
module:
```

**Comment by doronbehar** [comment]
Checked and it's still an issue.

**Comment by MetaDark** [comment]
This is no longer a problem with the fix applied in #123414:

https://github.com/NixOS/nixpkgs/pull/123414/files#diff-9b5cf33e6d36fc15bc1e54945cdaf1ac9be1e988bdc6cc91af70df5b1f81d075R31-R36.

```shell
> nix why-depends -f. texlab rustc
'texlab' does not depend on 'rustc'
```

**Comment by doronbehar** [comment]
Reopening now since the update to 4.0.0 at https://github.com/NixOS/nixpkgs/pull/175752 .
